xbmc/FileItem.h

```diff
@@ -555,6 +557,8 @@ class CFileItem : public CGUIListItem, public IArchivable, public ISerializable,
    */
   void FillMusicInfoTag(const std::shared_ptr<const PVR::CPVREpgInfoTag>& tag);
 
+  mutable std::optional<CURL> m_curlPath;
+  mutable std::optional<CURL> m_curlDynPath;
```

Comment:
May I suggest to rename the member to `m_urlDynPath` (omit the 'c') - reading the current name I was immediately thinking about a relationship to `libcurl` here...